src/components/notesCard/NotesCard.jsx

```diff
@@ -26,13 +30,18 @@ const NotesCard = (noteModal) => {
 
                 </div>
                 <div className="card__notes__actions__container flex jc-space-around">
-                    <FontAwesomeIcon icon={faArchive} />
-                    <FontAwesomeIcon icon={faRecycle} />
-                    <FontAwesomeIcon icon={faTrash} onClick={() => deleteFromNotes(dispatch_note, noteModal._id, token)} />
-                    <FontAwesomeIcon onClick={() => setIsModalOpenUpdate(true)} icon={faPen} />
+
+                    {archive || trash || <FontAwesomeIcon icon={faArchive} onClick={() => archiveHandler()} />}
+                    {archive || trash || <FontAwesomeIcon icon={faRecycle} />}
+                    {archive || trash || <FontAwesomeIcon onClick={() => setIsModalOpenUpdate(true)} icon={faPen} />}
```

Comment:
The app is getting crashed on editing the notes
Getting this error:-Uncaught TypeError: Cannot read properties of undefined (reading 'title')

Reply:
That was a typo, i changed a variable in props forgot to change it in modal file, That is Fixed now.